src/internal/components/expand-toggle-button/index.tsx

```diff
@@ -16,24 +16,45 @@ export function ExpandToggleButton({
   expandButtonLabel,
   collapseButtonLabel,
   customIcon,
+  invisible = false,
+  dataAttribute,
 }: {
   isExpanded?: boolean;
   onExpandableItemToggle?: () => void;
   expandButtonLabel?: string;
   collapseButtonLabel?: string;
   customIcon?: React.ReactNode;
+  invisible?: boolean;
+  dataAttribute?: { [key: string]: boolean };
 }) {
   const buttonRef = useRef<HTMLButtonElement>(null);
   const { tabIndex } = useSingleTabStopNavigation(buttonRef);
+
+  if (invisible) {
+    return (
+      <button
+        type="button"
+        ref={buttonRef}
+        tabIndex={tabIndex}
+        aria-label={isExpanded ? collapseButtonLabel : expandButtonLabel}
```

Comment:
I believe this should be aria-hidden instead, as the sole purpose of this button is to capture focus for keyboard nav.